src/runtime_src/xdp/profile/plugin/aie_trace/aie_trace_plugin.cpp

```diff
@@ -720,8 +720,9 @@ namespace xdp {
     }
 
     // New start & end events for trace control and counters
-    coreTraceStartEvent = XAIE_EVENT_TIMER_SYNC_CORE;
-	  coreTraceEndEvent   = XAIE_EVENT_TIMER_VALUE_REACHED_CORE;
+    //coreTraceStartEvent = XAIE_EVENT_TIMER_SYNC_CORE;
+    coreTraceStartEvent = XAIE_EVENT_TRUE_CORE;
```

Comment:
The reason why timer sync doesn't work is because we call reset *before* starting trace in fal. Since we use timer reset event as trace start, we should call reset *after* trace start. We should do it regardless of whether we use XAIE_EVENT_TIMER_SYNC_CORE or not.
Also need to verify the effect of using XAIE_EVENT_TRUE_CORE with Abraham

Reply:
I re-arranged timer reset to be after core start